test/kuttl/test-user-defined-ns/02-k8ssandra-cluster.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands: 
+- command:  kubectl create ns test-ns
+  ignoreFailure: false 
+- command:  kubectl apply -n test-ns --server-side --force-conflicts -f ../../testdata/samples/k8ssandra-single-kind.yaml
```

Comment:
Isn't this silly to test for failures if we intentionally apply with "do not care about failures, overwrite my mistakes" ? The --force-conflicts shouldn't probably be here since it masks issues.

Reply:
I thought `--force-conflicts` had something to do with server side apply and ensuring that even if some other controller was updating the resource, our mutation would still be applied? 

What issues does this mask? I don't follow your question I'm sorry.